### DISCUSSION TITLE
get mantine theme colors outside of component

### DISCUSSION THREAD

**Discussion by mlarcher** [mantinedev/mantine#2967] [Q&A]
Is there a way to access the Mantine theme colors outside of a component ? Afaict they're constants, and I'd like to use them in a configuration file for default colors

**Comment by rtivital** [comment]
You can import `DEFAULT_THEME` variable from `@mantine/core` package. Similar to this, you can create a variable where you store your theme override and import it from any file.

**Comment by mlarcher** [comment]
thanks for the quick reply

That would be nice to have this info somewher in the doc ;)
